src/models/controller.ts

```diff
@@ -0,0 +1,94 @@
+import express from 'express'
+const router = express.Router()
+import { v4 as uuid } from 'uuid'
+import { Request, Response, NextFunction } from 'express'
+import UserService from '../models/user'
+
+interface RequestBody {
+  name: string
+  age: number
+  id: string
+  path: string
+  nameOfFile: string
+}
+
+let user: UserService = new UserService()
+
+export const initializeUser = async (
+  req: Request<{}, {}, RequestBody, {}>,
+  res: Response
+) => {
+  try {
+    await user.init()
+    const { nameOfFile } = req.body
+    if (nameOfFile) {
+      res.status(200).send({ message: 'Dir/file is already exists' })
+    }
+    res.status(200).send({ message: 'Dir/file has been successfuly created' })
+  } catch (error) {
+    res.status(422).send({ error })
+  }
+}
+export const createUser = async (
+  req: Request<{}, {}, RequestBody, {}>,
+  res: Response
+) => {
+  try {
+    const { name, age } = req.body
```

Comment:
I think you need check that name field is not undefined. If name is undefined throw error. https://expressjs.com/ru/guide/error-handling.html
Also add for User entity separate fields: firstName and lastName